synapse/tests/test_lib_config.py

```diff
@@ -159,6 +164,24 @@ def __init__(self, proxy):
                     self.eq(tufo[1]['tufo:form'], 'inet:ipv4')
                     self.eq(tufo[1]['inet:ipv4'], 0)
 
+    def test_configable_noncallable(self):
+
+        with self.getRamCore() as core:
```

Comment:
Can we just invert the ``__init__`` logic in the above test with ``CoolClass`` to test this?

Reply:
Yep, you can simplify the test if you like. This is not the minimal scenario to reproduce the bug, but rather a realistic usage scenario.

Reply:
streamlined this a bit